source/operations/data-recovery/recover-after-site-failure.rst

```diff
@@ -1,13 +1,76 @@
 .. _minio-restore-hardware-failure-site:
 
-==========================
-Recover after Site Failure
-==========================
+=====================
+Site Failure Recovery
+=====================
 
 .. default-domain:: minio
 
 .. contents:: Table of Contents
    :local:
    :depth: 1
 
-ToDo- site replication recovery procedure
+MinIO can make the loss of an entire site, while significant, a relatively minor incident.
+Site recovery depends on the backup option you use for the site.
+
+Site Replication
+----------------
+
+:ref:`Site replication <minio-site-replication-overview>` keeps two or more MinIO deployments in sync.
```

Comment:
Can we be more specific? It keeps IAM, buckets, bucket configurations, and objects synchronized between sites.

Reply:
I did try to cover this with the replicable data link in the next sentence, but I moved things around.